qt_gui_cpp/src/qt_gui_cpp_sip/CMakeLists.txt

```diff
@@ -31,6 +31,39 @@ set(qt_gui_cpp_sip_INCLUDE_DIRS ${qt_gui_cpp_INCLUDE_DIRS} "${CMAKE_CURRENT_SOUR
 set(qt_gui_cpp_sip_LIBRARY_DIRS ${qt_gui_cpp_LIBRARY_DIRS} lib)
 set(qt_gui_cpp_sip_LDFLAGS_OTHER ${qt_gui_cpp_LDFLAGS_OTHER} -Wl,-rpath,\\"lib\\")
 
+find_library(class_loader_LIBRARY class_loader)
+find_library(console_bridge_LIBRARY console_bridge)
+find_library(TinyXML2_LIBRARY tinyxml2)
+
+get_filename_component(qt_gui_cpp_src_DIR ${CMAKE_CURRENT_BINARY_DIR} DIRECTORY)
+set(qt_gui_cpp_LIBRARY ${qt_gui_cpp_src_DIR}/qt_gui_cpp/Release/qt_gui_cpp${CMAKE_STATIC_LIBRARY_SUFFIX})
+
+set(_qt_gui_cpp_sip_LIBRARIES
+  ${PYTHON_LIBRARY}
+  ${ament_index_cpp_LIBRARIES}
+  ${rcutils_LIBRARIES}
```

Comment:
This is not a dependency of the package and it isn't being `find_package`d. So this variable is likely not even defined.

Reply:
This is the bare minimum to get things to link correctly. Printing out the environment variables tells me this was defined somewhere.

rcutils_LIBRARIES=/Users/brawner/workspace/ros2_core/install/rcutils/lib/librcutils.dylib

Changing this to pluginlib_LIBRARIES works.